index.html

```diff
@@ -491,7 +495,9 @@
 
       <a aria-hidden="true" href="#undo-circular" id="undo-circular" data-icon="undo-circular"></a>
 
-      <a aria-hidden="true" href="#undo" id="undo" data-icon="undo"></a>
+      <a aria-hidden="true" href="#undo-left" id="undo-left" data-icon="undo-left"></a>
+
+      <a aria-hidden="true" href="#undo-right" id="undo-right" data-icon="undo-right"></a>
```

Comment:
as I said in my comment, I don’t think the `undo` should be mirrored (counter-clockwise meaning).